CMakeLists.txt

```diff
@@ -125,7 +149,7 @@ aux_source_directory(src/exe EXE_SRC)
 # Define all sources
 set(DDPROF_GLOBAL_SRC ${COMMON_SRC} ${PPROF_SRC} ${EXPORTER_SRC} ${EXE_SRC})
 
-set(DDPROF_LIBRARY_LIST llvm-demangle ${ELFUTILS_LIBRARIES} Threads::Threads)
+set(DDPROF_LIBRARY_LIST llvm-demangle ${ELFUTILS_LIBRARIES} Threads::Threads DDProf::Parser)
```

Comment:
It is nice that we isolate this in it's own library
We should follow this pattern more. This way when we add new code, we don't recompile it many times.